simulation/traffic_simulator/src/math/hermite_curve.cpp

```diff
@@ -193,7 +203,8 @@ boost::optional<double> HermiteCurve::getCollisionPointIn2D(
       double poly_x = (1 - tx) * point1.x + tx * point0.x;
       double poly_y = (1 - ty) * point1.y + ty * point0.y;
       double error = std::hypot(poly_x - x, poly_y - y);
-      if (error < 3 && 0 < tx && tx < 1 && 0 < ty && ty < 1 && 0 < solution && solution < 1) {
+      /// @note Hard coded parameter, torelance of the collision point.
```

Comment:
Spell check warning should not be ignored: torelance -> tolerance.
![image](https://user-images.githubusercontent.com/1256980/129664414-56b96b81-9b6c-46c6-8ad2-2a14c9ec759b.png)